.storybook/components/Wrapper/Wrapper.js

```diff
@@ -13,7 +13,7 @@ class Wrapper extends Component<Props> {
     const { children } = this.props;
     return (
       <OuterWrapper>
-        <InnerWrapper>{children}</InnerWrapper>
+        <InnerWrapper> {children} </InnerWrapper>{' '}
```

Comment:
Was this change intentional?

Reply:
hum no, looks like a prettier bug 🤔

Reply:
Prettier escapes whitespaces in inline tags when it breaks them out to multiple lines to preserve proper spacing for `inline` level elements. For example

```js
<strong>Name:</strong> <span>{name}</span>
```

is changed to

```js
<strong>Name:</strong>{' '}
<span>{name}</span>
```

to ensure that it renders as **Name:**`<space>`Jane Doe and not **Name:** Jane Doe.

Unfortunately that occasionally causes unwanted escaping. Looks like that's what happened here.